src/main/java/cn/nukkit/Player.java

```diff
@@ -613,6 +616,7 @@ public Player(SourceInterface interfaz, Long clientID, String ip, int port) {
         this.chunkRadius = viewDistance;
         //this.newPosition = new Vector3(0, 0, 0);
         this.boundingBox = new SimpleAxisAlignedBB(0, 0, 0, 0, 0, 0);
+        this.lastSkinChange = -1;
```

Comment:
Why not set the value where it is declared?

Reply:
I thought I should do it with lastBreak it's the same.